x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_form.tsx

```diff
@@ -107,17 +108,53 @@ export function validateBaseProperties(alertObject: InitialAlert): ValidationRes
   return validationResult;
 }
 
-const hasErrors: (errors: IErrorObject) => boolean = (errors) =>
+export function getAlertErrors(
+  alert: Alert,
+  actionTypeRegistry: ActionTypeRegistryContract,
+  alertTypeModel: AlertTypeModel | null
+) {
+  const alertParamsErrors = (alertTypeModel
+    ? alertTypeModel.validate(alert.params).errors
+    : []) as IErrorObject;
```

Comment:
This looks wrong 🤔 
the type signature for `IErrorObject` can't be `[]`... this is probably passing because of the explicit `as` casting.
It's best to avoid this sort of casting as it can hide errors... better to do this:

```
const alertParamsErrors: IErrorObject =  ....
```